README.md

```diff
@@ -34,14 +34,14 @@ const settings = {
 export default settings;
 ```
 
-⚠️  Please do not present API credentials in public places.
+⚠️  Please do not display/use API credentials in public places/open WIFI.
```

Comment:
Well, this isn't the best change. I would rephrase the whole point into: Please keep your API credentials in a secret place.